doc/releases/release-notes-2.4.rst

```diff
@@ -265,6 +268,8 @@ Drivers and Sensors
   * Allow user to disable auto-start of IEEE 802.15.4 network interface.
     By default the IEEE 802.15.4 network interface is automatically started.
   * Added support for setting TX power in rf2xx driver.
+  * nordic: add 802.15.4 multiprotocol support, see :option:`CONFIG_NRF_802154_MULTIPROTOCOL_SUPPORT`.
```

Comment:
Looks a bit different than the other lines. Perhaps `* Added Nordic 802.15.4 multiprotocol ...`